puffin_egui/src/flamegraph.rs

```diff
@@ -150,7 +150,7 @@ impl Default for Options {
             frame_list_height: 48.0,
             frame_width: 10.0,
 
-            merge_scopes: true,
+            merge_scopes: false, // off, because it really only works well for single-threaded profiling
```

Comment:
I am currently unchecking this one each time I start puffin 🤦

Reply:
yep me too